### DISCUSSION TITLE
Help required for a gevent/greenlet error

### DISCUSSION THREAD

**Discussion by ayushpratap17** [ssahoo11742/KnightStable#4] [General]
Hi @Dodo11742 

I saw your post https://www.reddit.com/r/learnpython/comments/1003xd1/cant_understand_what_to_do_for_this_gevent_error/ but I don't see solution there. Were you able to resolve this issue? 

I'm getting "Cannot convert greenlet.greenlet to gevent._gevent_c_greenlet_primitives.SwitchOutGreenletWithLoop" error when trying to kill a greenlet spawned using gevent. Not sure what this error means. This error is occasionally seen.

Any help would be highly appreciated. 

Thank you!

**Comment by ssahoo11742** [comment]
On Tue, May 30, 2023 at 7:45 AM ayushpratap17 ***@***.***>
wrote:

> Hi @Dodo11742 <https://github.com/Dodo11742>
>
> I saw your post
> https://www.reddit.com/r/learnpython/comments/1003xd1/cant_understand_what_to_do_for_this_gevent_error/
> but I don't see solution there. Were you able to resolve this issue?
>
> I'm getting "Cannot convert greenlet.greenlet to
> gevent._gevent_c_greenlet_primitives.SwitchOutGreenletWithLoop" error when
> trying to kill a greenlet spawned using gevent. Not sure what this error
> means. This error is occasionally seen.
>
> Any help would be highly appreciated.
>
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SwayamSahoo11742/KnightStable/discussions/4>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A35SBIY7EWQNX7TBTI22TMTXIXMXLANCNFSM6AAAAAAYT4LDNQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
Hello,

I apologize I was not able to find a solution to this and ended up
abandoning trying that route.
It’s been a while, but I think I have a version of it where it ended up
working here’s the repo:
https://github.com/SwayamSahoo11742/KnightStable-product

I am not a 100% sure since it’s been a while (code’s probably outdated),
but I think those configs were good for me to deploy. I could be wrong

I hope you find success, if I find more I will let you know
